src/elli.erl

```diff
@@ -171,14 +176,14 @@ code_change(_OldVsn, State, _Extra) ->
 %%%===================================================================
 
 remove_acceptor(State, Pid) ->
-    State#state{acceptors = lists:delete(Pid, State#state.acceptors),
```

Comment:
All acceptor processes are linked to this server, so they will also stop. Since the acceptor processes are used for handling the requests the active requests will also be stopped. I guess this isn't a big deal for most users of elli, but some might find it surprising.